hivemind/compression/quantization.py

```diff
@@ -129,11 +127,19 @@ class BlockwiseQuantization(Quantization):
     compression_type = runtime_pb2.BLOCKWISE_8BIT
     codebook_dtype, indices_dtype = np.float32, np.uint8
 
+    def __init__(self):
+        super().__init__()
+
+        from bitsandbytes.functional import dequantize_blockwise, quantize_blockwise
```

Comment:
If it runs for the 2nd time, it won't actually import things again, just copy the references.